bump to Gluon v2025.1

T0biii opened the pull request:
wifi: mt76: mt76x02: wake queues after reconfig
mt7915: detect and purge stuck PLE queues
generic: disable kernel MPTCP support
gluon-core: use pubkey_privacy from provider
ath79-generic: (re)add support for Ubiquiti AirMax targets
ath79: add support for COMFAST CF-EW71 v2
core: fix ipv6 config for cellular devices

grische commented:
I think we want to switch to Tags again. And please change the commit message accordingly. 

I'll start renaming branches then.

T0biii commented:
Ah, I thought we were keeping it as Gluon main (it's just that 2025.1 is the last commit on main).
and create a new branch gluon-v2025.1.x

grische commented:
Effectively, yes. That's what I'd do. But the commit should mention the release branch it's based on and afaik it is gluon-v2025.1.x

grische committed:
bump to Gluon v2025.1

Also update the Batman V series for v2025.1, see
https://github.com/freifunkMUC/gluon/tree/v2025.1.x-batmanv

grische commented:
@T0biii if this builds, I'd rename the branch to gluon-v2025.1.x (before merging). And I would have no gluon-main until there is a need for it in the future again. What do you think?

grische commented:
We need updated modules before this can be merged:
```
# gluon-info
/usr/bin/lua: /usr/lib/lua/gluon/mesh-vpn.lua:39: attempt to call field 'pubkey_privacy' (a nil value)
stack traceback:
        /usr/lib/lua/gluon/mesh-vpn.lua:39: in function 'get_enabled_public_key'
        /usr/lib/lua/gluon/info.lua:11: in main chunk
        [C]: in function 'require'
        /usr/bin/gluon-info:2: in main chunk
        [C]: ?
```

Blocked by #757

grische reviewed (dismissed):
Blocked by #768

grische commented:
The new version "should" be working again. I did only some very basic testing on my AP, but apart from build changes I'd expect the core should be the same.

@T0biii do you want to try it as well before we merge it??

T0biii reviewed (commented):
cant submit an Approve but every thing seems to work fine for me
